content/en/docs/Tasks/creating-an-index.md

```diff
@@ -5,37 +5,123 @@ description: >
 ---
 
-## Prerequisites 
+## Prerequisites
 
-- [opm](https://github.com/operator-framework/operator-registry/releases) `v1.14.0+`
-- [docker](https://docs.docker.com/install/) version `17.03`+ or [podman](https://github.com/containers/libpod/blob/master/install.md) `v1.2.0+` or [buildah](https://github.com/containers/buildah/blob/master/install.md) `v1.7+`.
+- [opm](https://github.com/operator-framework/operator-registry/releases) `v1.19.0+`
 
+## Creating an Index
 
-# Creating an Index 
+`OLM`'s `CatalogSource` [CRD][catalogsource-crd] defines a reference to a catalog of operators that are available to
+install onto a cluster. To make your operator bundle available, you can add the bundle to a container image which the
+`CatalogSource` references. This image contains a record of bundle images that OLM can pull and extract the manifests
+from in order to install an operator.
 
-`OLM`'s `CatalogSource` [CRD][catalogsource-crd] define a reference to a catalog of operators that are available to install onto a cluster. To make your operator bundle available, you can add the bundle to a container image which the `CatalogSource` points to. This image contains a record of bundle images that OLM can pull and extract the manifests from in order to install an operator. 
+OLM uses a plaintext [file-based catalog][file-based-catalog-spec] format (JSON or YAML) to store these records in an index, and `opm` has tooling
+that helps initialize an index, render new records into it, and then validate that the index is valid. Let's walk
+through a simple example.
 
->Note: The container image also contains information that represents the upgrade graphs between different operator versions, an operator's dependencies etc graphically. To learn more about the upgrade graph of an operator, checkout the [creating an upgrade graph doc][upgrade-graph-doc]  
+First, we need to initialize our index, so we'll make a directory for it, generate a Dockerfile that can build an index
+image, and then populate our index with our package definition.
 
-So, to make your operator available to OLM, you can generate an index image via opm with your bundle reference included:
 
+### Initialize the index
 ```sh
-$ opm index add --bundles quay.io/my-container-registry-namespace/my-manifest-bundle:0.0.1 --tag quay.io/my-container-registry-namespace/my-index:1.0.0
-$ podman push quay.io/my-container-registry-namespace/my-index:1.0.0
+$ mkdir example-operator-index
+$ opm alpha generate dockerfile example-operator-index
+$ opm init example-operator \
+    --default-channel=preview \
+    --description=./README.md \
+    --icon=./example-operator.svg \
+    --output yaml > example-operator-index/index.yaml
 ```
 
-The resulting image is referred to as an `Index`. Now that image is available for clusters to use and reference with `CatalogSources` on their cluster.
+Let's validate our index to see if we're ready to ship!
+```sh
+$ opm validate example-operator-index
+FATA[0000] invalid index:
```

Comment:
Probably should change this error message in opm too to say catalog instead of index.